src/script/script_config.hpp

```diff
@@ -139,7 +139,7 @@ class ScriptConfig {
 	/**
 	 * Randomize all settings the Script requested to be randomized.
 	 */
-	virtual void AddRandomDeviation();
+	virtual void AddRandomDeviation(bool all = true);
```

Comment:
Missing documentation comment update.

Reply:
Specifically, it should use the doxygen `@param all ...` format